src/lib/mongodb.ts

```diff
@@ -14,6 +14,10 @@ declare global {
   }
 }
 
+const globalWithMongo = global as typeof globalThis & {
+  _mongoClientPromise: Promise<typeof MongoClient>
+}
+
```

Comment:
This used to be. I'm not sure when that was removed and I'll admit I'm ignorant of which is better / more common pattern. @abbyoung or @jcbcapps do you have any thoughts?

```typescript
// Required to use global._mongoClientPromise
declare global {
  /* eslint-disable-next-line */
  namespace NodeJS {
    interface Global {
      _mongoClientPromise: Promise<typeof MongoClient>
    }
  }
}
```